Escaped values﻿ not handled

The action should handle [escaped values](https://www.jetbrains.com/help/teamcity/build-script-interaction-with-teamcity.html#Escaped+values).

This message was cut before useful part
 
`##[error]##teamcity[testFailed name='testDescribeJson' message='PDOException : SQLSTATE|[42000|]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near |'(3) DEFAULT CURRENT_TIMESTAMP(3),|' at line 9' details=' /home/runner/file.php:72|n ' duration='4' flowId='9424']`

And reported as `PDOException : SQLSTATE|[42000|]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near |`

--- a/index.js
+++ b/index.js
@@ -2,7 +2,7 @@ const fs = require('fs');
 
 const matchers = {
   "phpunit-failure": {
-      "regexp": "##teamcity\\[testFailed.+message='([^']+)'.+details='\\s+{{GITHUB_WORKSPACE}}/([^:]+):(\\d+)[^']+'",
+      "regexp": "##teamcity\\[testFailed.+message='(.+)'.+details='\\s+{{GITHUB_WORKSPACE}}/([^:]+):(\\d+)[^']+'",
       "defaultSeverity": "error",
       "message": 1,
       "file": 2,

